### ISSUE TITLE
Fix problem with slim and batch normalization during test phase

### ISSUE THREAD

**Issue by davidsandberg** [davidsandberg/facenet#56]
There seems to be a problem with how the slim implementation of batch norm behaves when is_training==False. As a work-around is_training is set to True also when embeddings are calculated for evaluation on LFW.

**Comment by apollo-time** [comment]
So need setting is_training to True when calculate embedding of  new face?
If phase_train set to True, and try twice calculate embedding of one face, I got different embedding result.
How can I use this graph in real face feature application?

**Comment by davidsandberg** [comment]
This has been fixed by setting "updates_collections" in batch_norm_params to None to enable in-place updates of the mean and variance estimates in the batch norm. Also the decay was decreased to speed up the updates of the mean and variance estimates.
